app/models/court.rb

```diff
@@ -132,7 +132,7 @@ def add_uuid
 
   def fetch_image_file
     if image.present?
-      self.image_file.download!("https://hmctscourtfinder.justice.gov.uk/courtfinder/images/courts/#{self.image.to_s}")
+      self.image_file.download!("https://courtfinder-servicegovuk-production.s3.amazonaws.com/images/#{self.slug.underscore}.jpg")
```

Comment:
of course, the *right* thing to do here would be to replace the hostname with an environment variable, but I think at the moment we're in just-do-absolute-minimal-changes-needed-to-get-it-working mode, so in this case I think it's OK